server/src/MessageHandler.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  TextDocumentPositionParams,
+  CancellationToken,
+  TextDocuments,
+  DocumentFormattingParams,
+  TextEdit,
+  Range,
+  Location,
+  DeclarationParams,
+  CompletionParams,
+  CompletionItem,
+  CompletionItemKind,
+} from 'vscode-languageserver'
+import * as util from './util'
+import { fullDocumentRange } from './provider'
+import { getDMMF } from '@prisma/sdk'
+import { TextDocument, Position } from 'vscode-languageserver-textdocument'
+import format from './format'
+import * as os from 'os'
+
+export class MessageHandler {
+  constructor() {}
+
+  async handleHoverRequest(
+    params: TextDocumentPositionParams,
+    documents: TextDocuments<TextDocument>,
+  ) {
+    return null
+  }
+
+  getWordAtPosition(document: TextDocument, position: Position): String {
+    let currentLine = document.getText({
+      start: { line: position.line, character: 0 },
+      end: { line: position.line, character: 9999 },
+    })
+    // search for the word's beginning and end
+    var beginning = currentLine.slice(0, position.character + 1).search(/\S+$/)
+    var end = currentLine.slice(position.character).search(/\W/)
+    if (end < 0) {
+      return ''
+    }
+    return currentLine.slice(beginning, end + position.character)
+  }
```

Comment:
```suggestion
  getWordAtPosition(document: TextDocument, position: Position): string {
    const currentLine = document.getText({
      start: { line: position.line, character: 0 },
      end: { line: position.line, character: 9999 },
    })
    // search for the word's beginning and end
    const beginning = currentLine.slice(0, position.character + 1).search(/\S+$/)
    const end = currentLine.slice(position.character).search(/\W/)
    if (end < 0) {
      return ''
    }
    return currentLine.slice(beginning, end + position.character)
  }
```

In TypeScript there is a difference between `string` and `String`
Short answer always use the lowercase one 😄 More info at https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types